common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/ElasticSearchException.scala

```diff
@@ -0,0 +1,40 @@
+package com.gu.mediaservice.lib.elasticsearch
+
+import com.gu.mediaservice.lib.logging.{LogMarker, MarkerMap}
+import com.sksamuel.elastic4s.ElasticError
+import net.logstash.logback.marker.LogstashMarker
+
+trait ElasticSearchError {
+  self: Throwable =>
+  def error: ElasticError
+
+  def markerContents: Map[String, Any]
+}
+
+object ElasticSearchException {
```

Comment:
does this need to extend `LogMarker`?

Reply:
No, because it has an unapply that gives us a `LogMarker`